### PR TITLE
upgrade to 0.15.0

### PR DESCRIPTION
**Pull request by ramfox**
does not include `iroh-s3-bao-store`

### CODE REVIEW

**Comment by dignifiedquire** [comment]
sad ci

File: dumbpipe-web/Cargo.lock [hunk 1]
```diff
@@ -789,7 +789,7 @@ dependencies = [
  "anyhow",
  "clap",
  "hex",
- "iroh-net",
+ "iroh-net 0.14.0",
```
Comment (dignifiedquire) [inline dumbpipe-web/Cargo.lock:792]: this still has an `0.14` dependency?

Comment (ramfox) [reply dumbpipe-web/Cargo.lock:792]: ???? sigh, i probably accidentally did a key stroke or something

Comment (dignifiedquire) [reply dumbpipe-web/Cargo.lock:792]: note: this is a lock file, this just means sth in the dep tree still depends on 0.14.0

Comment (ramfox) [reply dumbpipe-web/Cargo.lock:792]: yea, we need to wait until the latest dumbpipe is released on crates.io , created an issue #51

**Review by dignifiedquire — APPROVED** [review]

**Pull request merged**